### PR TITLE
Feature 2242 download members

### PR DESCRIPTION
**Pull request by mvolkmann**

### CODE REVIEW

File: web-ui/src/components/admin/users/Users.jsx [hunk 1]
```diff
@@ -99,6 +125,14 @@ const Users = () => {
                 <Button startIcon={<PersonIcon />} onClick={handleOpen}>
                   Add Member
                 </Button>
+                {selectHasProfileReportPermission(state) && (
```
Comment (mvolkmann) [inline web-ui/src/components/admin/users/Users.jsx:128]: This part adds the new button if the user has the required permission.

File: web-ui/src/components/menu/Menu.jsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import React, { useContext, useEffect, useRef, useState } from 'react';
 
 import { postEmployeeHours } from '../../api/hours';
-import { reportAllMembersCsv } from '../../api/member';
```
Comment (mvolkmann) [inline web-ui/src/components/menu/Menu.jsx:4]: All the deleted lines moved to `Users.jsx` above.

File: web-ui/src/context/selectors.js [hunk 3]
```diff
@@ -84,6 +84,16 @@ export const selectHasCheckinsReportPermission = createSelector(
     )
 );
 
+export const selectHasProfileReportPermission = createSelector(
```
Comment (mvolkmann) [inline web-ui/src/context/selectors.js:87]: This is a new permission check that is used in `Users.jsx`.

**Review by S78901 — APPROVED** [review]
Looking good!

**Review by mkimberlin — APPROVED** [review]

**Pull request merged**